src/wp-includes/revision.php

```diff
@@ -516,6 +516,51 @@ function wp_get_post_revisions( $post_id = 0, $args = null ) {
 	return $revisions;
 }
 
+/**
+ * Returns last revisions of specified post and count of total revisions.
+ *
+ * @since 6.1.0
+ *
+ * @param int|WP_Post $post_id Optional. Post ID or WP_Post object.
```

Comment:
```suggestion
 * @param int|WP_Post $post Optional. Post ID or post object. Default is the global $post.
```

This matches the variable name and description for `get_permalink()`, `get_the_title()` and many other functions with a similar signature.